content/spin/v2/triggers.md

```diff
@@ -69,11 +74,57 @@ These ways of writing components achieve the same thing, so which should you cho
 Named components have the following advantages:
 
 * Reuse. If you want two triggers to behave in the same way, they can refer to the same named component. Remember this means they are not just handled by the same Wasm file, but with the same settings.
-* Named. If an error occurs, Spin can tell your the name of the component where the error happened. With inline components, Spin has to synthesize a name. This isn't a big deal in single-component apps, but make diagnostics harder in larger apps.
+* Named. If an error occurs, Spin can tell you the name of the component where the error happened. With inline components, Spin has to synthesize a name. This isn't a big deal in single-component apps, but makes diagnostics harder in larger apps.
 
 Inline components have the following advantages:
 
 * Compact, especially when using inline table syntax.
 * One place to look. Both the trigger event and the handling details are always in the same piece of TOML.
 
 If you are not sure, or are not experienced, we recommend using named components at first, and adopting inline components as and when you find cases where you prefer them.
+
+### Setting Up Multiple Trigger Types
+
+In this section, we build an application that contains multiple triggers.
+
+> Note: Not all templates support the `spin add` command. In particular, at the time of writing, none of the default Redis templates support being added to existing applications. Therefore, if you want to use `spin add` to build an application with both Redis and HTTP triggers, you should first create a Redis application, then use `spin add` to add HTTP triggers, as shown below. (You won't be able to add additiona Redis triggers this way; if you need those, you'll need to set them up manually for now.)
+
+Here is an example of creating an application with both HTTP and Redis triggers:
+
+<!-- @nocpy -->
+
+```bash
+# Start with a Redis trigger application
+$ spin new -t redis-rust trigger-example
+Description: A multiple trigger example
+Redis address: redis://localhost:6379
+Redis channel: one
+# Change into to the application directory
+$ cd trigger-example 
+# Create HTTP trigger application
+$ spin add -t http-rust http-trigger-example  
+Description: A HTTP trigger example
+HTTP path: /...
+```
+
+The above `spin new` and `spin add` commands will automatically scaffold a Spin manifest (`spin.toml` file) similar to the following:
```

Comment:
Should we indicate more explicitly that this is only the "interesting" bit of the manifest, e.g.
 
```suggestion
The above `spin new` and `spin add` commands will scaffold a Spin manifest (`spin.toml` file) with the following triggers:
```

Reply:
Oooh, yes, I like that!